ground/gcs/src/plugins/boards_dronin/sprf3econfiguration.h

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
This is a own family of boards, so it should have it's own configuration form. So we are possible to place some text hints for configuring or a warning like "experimental support" in the first time.